examples/document_splitter_schema.py

```diff
@@ -18,7 +19,7 @@
 class Request(BaseRequest):
     __root__: List[Text]
 
-    def transform(self):
+    def transform(self) -> Any:
```

Comment:
why are we doing json dumps here?
I recall that we removed this in our [recent schemas](https://github.com/mietright/conny-ml-classification/blob/main/conny_ml/classification/schemas/giro_document_splitting.py)
because we simplified the corresponding code in the schema handler: https://github.com/connylabs/meowlflow/blob/204d1255d01fef4e6c880ed646b1d0938a1b1ebf/meowlflow/sidecar.py#L172

Reply:
i think i recall that we had decided against the de/serialization when using the `serve` approach since it was superfluous: the pyfunc didnt need to receive a serialized payload (and the deserialize it) if the wholeapp runs on the same endpoint.

i think this de/serialization is only necessary when using a sidecar?

Reply:
Yes I spent way too much time on Friday only to re-learn that in the evening. Let me push my latest work, which actually raises some more questions.

Tldr, we indeed luckily don't need to de/serialize the request but in some cases we do have to re-shape it. I created CI that uses the mlflow example model [0] and I can only make it work if the schema transforms the request into an actual pandas dataframe object; a dict that has the same-ish shape as a dataframe isn't good enough. It seems like some models are very strict about what they accept.

[0] https://github.com/mlflow/mlflow-example

Reply:
wow. very tricky!